firefox-ios/Client/metrics.yaml

```
@@ -709,6 +709,27 @@ app:
       - fx-ios-data-stewards@mozilla.com
     expires: "2025-01-01"
 
+# DAU reporting
+usage:
+  profile_id:
+    type: uuid
+    lifetime: user
+    description: |
+      A UUID uniquely identifying the profile,
+      not shared with other telemetry data.
+    bugs:
+      - https://github.com/mozilla-mobile/firefox-ios/pull/
+    data_reviews:
+      - https://github.com/mozilla-mobile/firefox-ios/pull/
+    data_sensitivity:
+      - technical
+      - highly_sensitive
+    notification_emails:
+      - fx-ios-data-stewards@mozilla.com
```

Can you add `glean-team@mozilla.com` here too?
```suggestion
      - fx-ios-data-stewards@mozilla.com
      - glean-team@mozilla.com
```